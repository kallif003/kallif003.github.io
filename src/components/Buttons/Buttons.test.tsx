import * as React from "react"
import { render } from "@testing-library/react"

import {
	Button1,
	Button2,
	Button3,
	Button4,
	Button5,
	Button6,
	Button7,
	Button8,
} from "."

describe("Buttons", () => {
	it("renders Button1 properly", () => {
		const { container } = render(<Button1>Teste</Button1>)
		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders Button2 properly", () => {
		const { container } = render(<Button2>Teste</Button2>)
		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders Button3 properly", () => {
		const { container } = render(<Button3>Teste</Button3>)
		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders Button4 properly", () => {
		const { container } = render(<Button4>Teste</Button4>)
		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders Button5 properly", () => {
		const { container } = render(<Button5>Teste</Button5>)
		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders Button6 properly", () => {
		const { container } = render(<Button6>Teste</Button6>)
		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders Button7 properly", () => {
		const { container } = render(<Button7>Teste</Button7>)
		expect(container.firstChild).toMatchSnapshot()
	})
	it("renders Button8 properly", () => {
		const { container } = render(<Button8>Teste</Button8>)
		expect(container.firstChild).toMatchSnapshot()
	})
})
