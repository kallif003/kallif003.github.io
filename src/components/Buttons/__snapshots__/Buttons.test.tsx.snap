// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buttons renders Ancora properly 1`] = `
<a
  class="sc-jrQzAO hLTNwg"
>
  Teste
</a>
`;

exports[`Buttons renders Button1 properly 1`] = `
<button
  class="sc-bdvvtL hQRTiU"
>
  Teste
</button>
`;

exports[`Buttons renders Button2 properly 1`] = `
<button
  class="sc-gsDKAQ dWvCyB"
>
  Teste
</button>
`;

exports[`Buttons renders Button3 properly 1`] = `
<button
  class="sc-dkPtRN EaVla"
>
  Teste
</button>
`;

exports[`Buttons renders Button4 properly 1`] = `
<button
  class="sc-hKwDye iThMGh"
>
  Teste
</button>
`;

exports[`Buttons renders Button5 properly 1`] = `
<button
  class="sc-eCImPb bFFGJs"
>
  Teste
</button>
`;

exports[`Buttons renders Button6 properly 1`] = `
<button
  class="sc-jRQBWg fYwWpf"
>
  Teste
</button>
`;

exports[`Buttons renders Button7 properly 1`] = `
<button
  class="sc-gKclnd dGTRWn"
>
  Teste
</button>
`;

exports[`Buttons renders Button8 properly 1`] = `
<button
  class="sc-iCfMLu hvSlZD"
>
  Teste
</button>
`;

exports[`Buttons renders Li properly 1`] = `
<li
  class="sc-pVTFL fBDvNW"
>
  Teste
</li>
`;

exports[`Buttons renders Ul properly 1`] = `
<ul
  class="sc-furwcr hopZPv"
>
  Teste
</ul>
`;
